app/src/main/java/org/mozilla/fenix/collections/CollectionCreationTabListAdapter.kt

```diff
@@ -45,9 +45,9 @@ class CollectionCreationTabListAdapter(
                 is CheckChanged -> {
                     val checkChanged = payloads[0] as CheckChanged
                     if (checkChanged.shouldBeChecked) {
```

Comment:
Hi, this is just a drive-by comment: 
If you decided to drop true/false and use checkChanged.shouldBeChecked to call holder::changeCheckBoxState, why not also change the if condition?
I was thinking of something like checkChanged.souldBeChecked || checkChanged.shouldBeUnchecked, it is also easier to read.

Reply:
You are right. Thanks